### PR TITLE
Milestone 3: Setup associations

### PR DESCRIPTION
**Pull request by asharanjith**
In this PR

- [x] Create a class Classroom, Book, and Rental class
- [x] Set up the relationship between Classroom and Student
- [x] Set up relationship between Person and Book using the intermediate class Rental

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -22,6 +23,11 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
+
```
Comment (MeqdamAlqudah) [inline person.rb:30]: - [  ] Please Getters and Setters for @rentals should be present you can add the rentals to attr_accessor also

File: student.rb [hunk 2]
```diff
@@ -6,7 +6,14 @@ def initialize
     @classroom = classroom
```
Comment (MeqdamAlqudah) [inline student.rb:7]: - [ ] Please add classroom as a parameter here it gives this error 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @asharanjith , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Highlights
:+1: Readme file 
:+1: no linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @asharanjith, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. Perfect implementation based on review 💯
2. All requirements are complete 💯

## Optional suggestions
You are free to remove this line from your readme:

> NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**